cypress/tests/inventory-card.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { DATA_FEDERATION_NAMESPACE, MINUTE, SECOND } from '../constants/common';
+import {
+  DATA_SOURCE_INPUTS,
+  Providers,
+  secretInputs,
+  SINGLE_BUCKET_POLICY,
+  TEST_DATA_SOURCE,
+} from '../constants/tests';
+import { BPCommon } from '../views/bucket-policies';
+import { app } from '../views/common';
+import { DSCommon } from '../views/data-resource';
+import { MCGMSCommon } from '../views/mcg-ms-common';
+
+describe('admin dashboard - inventory card', () => {
+  const PROCESSING_DS = 'e2e-processing-data-source';
+
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    MCGMSCommon.visitMcgMsDashboard();
+    app.waitForLoad();
+  });
+
+  after(() => {
+    MCGMSCommon.visitMcgMsDashboard();
+    MCGMSCommon.visitBucketPolicyList();
+    BPCommon.deleteFromListPage(SINGLE_BUCKET_POLICY);
+    MCGMSCommon.visitDataSourceListPage();
+    DSCommon.deleteFromListPage(PROCESSING_DS);
+    cy.wait(30 * SECOND);
```

Comment:
this 30 sec wait is after deleting bucketclass.... and before deleting corresponding data source (noobaa takes some time to get things reflected on it's end clearly: https://github.com/red-hat-storage/mcg-ms-console/blob/0f28bc302b80452a114684a67d1750328745f9a5/cypress/tests/bucket-policy.spec.ts#L26)...
also we can not do it in `beforeEach` @alfonsomthd otherwise it will wait for 30 secs everytime, before every `it`...

consistency issue mentioned on different thread I agree is true... we should not mix 2 `it` together... if needed we can merge the 2 `it` blocks...

Reply:
@SanjalKatiyar I suggested the `before` hook, not `beforeEach`.

Reply:
I was thinking of having a context or describe block related to particular tests inside the main describe block.